sdk/helper/testcluster/docker/environment.go

```diff
@@ -470,10 +470,12 @@ type DockerClusterNode struct {
 	Logger               log.Logger
 	cleanupContainer     func()
 	RealAPIAddr          string
+	ContainerID          string
```

Comment:
Isn't this redundant with the Container field?

Reply:
Yeah fair, I suppose Container.ID is simple enough.